models/_generate_layers.py

```diff
@@ -496,6 +497,46 @@ def collect_layers():
   return ls
 
 
+def get_super_call_kwargs_repr(layer: LayerBase) -> str:
```

Comment:
Why do you make this as a separate function, and not a member function of `LayerSignature`?

Reply:
And additionally, refactor and call it `get_super_call_code_str`, as I suggested above.